bin/fuel-core/src/cli/run.rs

```diff
@@ -306,25 +311,32 @@ impl Command {
             info!("Block production disabled");
         }
 
-        let consensus_key = load_consensus_key(consensus_key)?;
-        if consensus_key.is_some() && trigger == Trigger::Never {
+        let consensus_signer = if let Some(key_id) = consensus_aws_kms {
+            let config = aws_config::load_from_env().await;
+            let client = aws_sdk_kms::Client::new(&config);
+            // Ensure that we can use the key
+            let _ = client.get_public_key().key_id(&key_id).send().await?;
```

Comment:
This call will tell you if the key you're trying to access is valid & you can access the public key, but it won't tell you if you actually have permission to use it to sign payloads.

The only way to tell if you can sign payloads is to actually try to do that, which I would not recommend. But may be good to clarify the comment so future readers know what's going on here.